### PR TITLE
Show menu on mobile & other tweaks

### PR DESCRIPTION
**Pull request by PhieF**
Just a few tweaks for mobile devices, 

I've brought the changes I've made here, as we've talked before

https://github.com/PhieF/FreshRssExtensions/tree/master/xExtension-Androidify

### CODE REVIEW

File: p/themes/base-theme/template.css [hunk 1]
```diff
@@ -845,13 +845,15 @@ pre.enclosure-description {
...
 	.header,
-	.aside .btn-important,
 	.flux_header .item.website span,
 	.item.date, .day .date,
 	.dropdown-menu > .no-mobile,
 	.no-mobile {
 		display: none;
 	}
+	.aside .btn-important{
+		margin-top:10px;
```
Comment (Frenzie) [inline p/themes/base-theme/template.css]: nitpick

File: p/themes/base-theme/template.css [hunk 2]
```diff
@@ -845,13 +845,15 @@ pre.enclosure-description {
...
 @media(max-width: 840px) {
 	.header,
-	.aside .btn-important,
 	.flux_header .item.website span,
 	.item.date, .day .date,
 	.dropdown-menu > .no-mobile,
 	.no-mobile {
 		display: none;
 	}
+	.aside .btn-important{
```
Comment (Frenzie) [inline p/themes/base-theme/template.css]: nitpick

File: p/themes/Origine/origine.css [hunk 3]
```diff
@@ -1058,4 +1060,25 @@ a.btn {
 	.notification a.close .icon {
 		display: none;
 	}
+	.post {
+        padding-left: 10px;
+        padding-right: 10px;
+    }
+    .stick {
+        max-width: calc(100% - 87px);
```
Comment (Frenzie) [inline p/themes/Origine/origine.css]: What is the fallback?

**Comment by Alkarex** [comment]
Thanks @PhieF :-)

**Comment by Alkarex** [comment]
* [x] Minor whitespace fixes done https://github.com/FreshRSS/FreshRSS/pull/1929/commits/94b87bd38ca1ad7fdcea48896e65fb2ec1160cb8
    * [ ] Remains one Frenzie comment
* [ ] Need to check the other themes
* [ ] Check if the SVG icon for the menu is needed, or if the Unicode ⋮ is enough
* [ ] Testing

**Comment by Alkarex** [comment]
For reference, the same view before the changes:
image

**Comment by PhieF** [comment]
hm weird, what theme do you use ?
do you have the head bar usually ? before my commits

**Comment by Alkarex** [comment]
I just tested with *Origine*

**Comment by PhieF** [comment]
Grumpf also testing on this one, do you use to have the deconnection button, before ?
looks like the head bar is empty

**Comment by PhieF** [comment]
Ah that may explain why, I suppose the whole header bar is hidden in that case

**Comment by PhieF** [comment]
(completely fixed, I guess)

**Comment by Alkarex** [comment]
Looks better 👍 
For reference:

With the theme *Origine*:

image


With the theme *Pafat*:
 
image

**Comment by PhieF** [comment]
strange for the second one, but still usable, I think forcing the icon to the left can be dangerous

**Comment by PhieF** [comment]
Two more commits : a fix on scrolling with the "mobile ui"

and a suggestion: on mobile, focus on article content when selecting a title, what to you think ?

**Comment by Frenzie** [comment]
@PhieF 
> and a suggestion: on mobile, focus on article content when selecting a title, what to you think ?

How do you mean precisely?

**Comment by Alkarex** [comment]
Sorry for the delay on this PR. When I tested it, there were a few problems such as slight performance issues, and I got sidetracked. Let's come back to it after https://github.com/FreshRSS/FreshRSS/pull/2117

**Comment by marienfressinaud** [comment]
I'm closing this PR in favour of #2938, there were too many conflicts to be reusable.

**Pull request closed without merging**